meta-balena-dunfell/recipes-connectivity/bluez5/bluez5_%.bbappend

```diff
@@ -0,0 +1,2 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+SRC_URI_remove = "file://run-bluetoothd-with-experimental-flag.patch"
```

Comment:
I moved the patch to thud and warrior, is there an obvious way to tell if we need anything older based on when it was introduced to common?

I also made the new service file common (rather than dunfell only) and copied the old service file to thud and warrior.

Reply:
I made a quick search and the following device families are in older Yocto releases:
* Pyro: jetson-tx1, artik, jetson-tx2
* Rocko: xilinx, iot2000
* Sumo: odroid, val100, ts, compulab, asus-tinker-board, up-board
You could ask the devices team to set you up a test system to try out one representative of each, or at least open a ticket on their respective repositories to run specific test cases when running the release validation on these.

Reply:
Thanks for the feedback! I pushed all the related changes to the remaining compatibility layers and I'm working on compiling the older device repositories for testing now.